packages/node_modules/@webex/plugin-meetings/src/media/index.js

```diff
@@ -54,7 +54,7 @@ const Media = {};
  */
 Media.setLocalTrack = (enabled, track) => {
   if (track) {
-    track.enabled = !enabled;
+    track.enabled = enabled;
```

Comment:
this is not changing any logic, because I've changed all the places where setLocalTrack() was called to invert the value of enabled. I just did this, because the existing code was confusing: using "enabled" to mean "disabled"